zulip/zulip/examples/update-flags

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
```

Comment:
After merging this, I kinda think this should be `update-message-flags`, for clarity.